www/src/components/Settings.tsx

```diff
@@ -43,7 +46,7 @@ const Settings = () => {
           <Form.Control
             as="select"
             value={settings.theme}
-            onChange={(e) => handleSettingsChange('theme', e.target.value)}
+            onChange={(e: { target: { value: string; }; }) => handleSettingsChange('theme', e.target.value)}
```

Comment:
More suitable maybe `e: React.ChangeEvent<HTMLInputElement>`?

Reply:
You are right, thanks